python/vTestInterface.py

```diff
@@ -174,11 +173,26 @@ def generateTestInfo(enviroPath, test):
 # knowledge of "testabilty"
 globalListOfTestableFunctions = []
 
-# The extension needs to know if the environment was built
-# with mocking support, not just if the tool supports it
-# If the enviro was not built with mocking then the new mocking
-# fields in the API will be set to None by the migration process
-enviroSupportsMocking = None
+
+def getEnviroSupportsMock(enviroPath):
+    """
+    The extension needs to know if the environment was built with mocking
+    support, not just if the tool supports it.
+
+    If the enviro was not built with mocking then the new mocking fields in the
+    API will be set to None by the migration process.
+    """
+    try:
```

Comment:
Are you assuming that the call to get the API will do the migrate?  IF so add that as a comment.

Note I have a bug right now where the data api is not migrating the cover db version

Reply:
As discussed; this is not being addressed as how/when DataAPI/CoverDB migrates is not standardised.